vscode-extension/src/diagnostic.ts

```diff
@@ -68,7 +69,7 @@ function formatPlanstToDiag(plansStr: string, log: Logger): vscode.Diagnostic[]
 
     let diagnostics: vscode.Diagnostic[] = [];
     plans.forEach(
-        (plan: any) => {
+        (plan: UpgradePlan) => {
```

Comment:
```suggestion
        plan => {
```
and line 61 should be `let plans: Plan[]`;